CMakeLists.txt

```diff
@@ -152,14 +152,14 @@ add_subdirectory(examples)
 # Create helper method which adds relevent cargo tests for a given manifest
 function(add_test_cargo TEST_NAME_PREFIX MANIFEST_PATH ADD_DOCTESTS)
     # Add cargo as a test
-    add_test(NAME ${TEST_NAME_PREFIX}_cargo_tests COMMAND cargo test --all-targets --all-features --manifest-path ${MANIFEST_PATH})
+    add_test(NAME ${TEST_NAME_PREFIX}_cargo_tests COMMAND cargo test --all-targets --manifest-path ${MANIFEST_PATH})
```

Comment:
unfortunate that we need to disable `--all-features`, as later we might have multiple features in CXX-Qt we want to test :thinking: maybe OK for now though.